src/url-params.ts

```diff
@@ -189,16 +195,29 @@ export class AnalyticsUrlParams {
 			actualDomainMatch = null;
 		}
 
-		const destinationDomainMatch = destinationUrl
-			? destinationUrl.hostname.match(regex)
-			: null;
+		actualDomain = actualDomainMatch ? actualDomainMatch.toString() : null;
```

Comment:
```suggestion
		const actualDomain = actualDomainMatch ? actualDomainMatch.toString() : null;
```
and if you have recent enough TS in this project, then you can do:
```
		const actualDomain = actualDomainMatch?.[0]; // this is inferred as `string | null` automatically
```
prefer explicitly reading the first match, that relying on `toString()`.